core/inbox_test.go

```diff
@@ -408,6 +409,19 @@ func (p *fakeChainProvider) GetBlock(ctx context.Context, cid cid.Cid) (*types.B
 	return &blk, nil
 }
 
+func (p *fakeChainProvider) GetTipSet(tsKey types.SortedCidSet) (types.TipSet, error) {
+	ctx := context.TODO() // Should GetTipSet require a context everywhere?
+	var blocks []*types.Block
```

Comment:
I think we can reuse `chain.LoadTipsetBlocks` here without any issue.

Reply:
Thanks, I intended that and forgot :-)